native/ax/StrmDeMux/SDeMuxFilter.cpp

```diff
@@ -2414,7 +2414,8 @@ void CSDeMuxFilter::dump_data( char* pData, int dwBytes )
 void CSDeMuxFilter::dump_sample( IMediaSample *pSample )
 {
 	BYTE* pBuffer;
-	int   Length;
+	size_t Length = 0;
+	size_t offset = 0;

```

Comment:
Change 'offset' in the header...or just get rid of it altogether since it doesn't look like it's used for anything.

Reply:
Line 2424 complains of size_t to int conversion for offset

Changed in header